app.py

```diff
@@ -1,15 +1,42 @@
 import openai
 import streamlit as st
+from streamlit_chat import message
+import pandas as pd
 
 # Setting page title and header
-st.set_page_config(page_title="waam", page_icon=":robot_face:")
+st.set_page_config(page_icon=":bulb:", page_title="WAAP-GPT")
+st.markdown("<div style='text-align: center;'><h1 style='display: inline-block;'> 💡WAAM-GPT</h1><h5 style='display: inline-block; margin-left: 10px; color: gray;'>homework help</h5></div>", unsafe_allow_html=True)
 
 # Set org ID and API key
 openai.organization = st.secrets["openai_org"]
 openai.api_key = st.secrets["openai_key"]
 
 system_prompt = "You are a waam, a helpful large language model STEM tutor created during the 2023 5C Hackathon. You help users learn quantitative skills by guiding them through concepts and practice problems step by step instead of immediately giving away the final answer. Never give a student the direct answer. Always use markdown for your responses. Always render equations using LaTeX."
 
+# Autogenerate message
+st.write("Welcome to WAAP! We are here to help you do well in school")
```

Comment:
It is called waam, not WAAP lol. Lets also make the message more specific saying things like I can teach you STEM topics.